java-client/src/main/java/org/opensearch/client/opensearch/core/BulkRequest.java

```diff
@@ -580,15 +550,6 @@ public BulkRequest build() {
 					buf.append("/_bulk");
 					return buf.toString();
 				}
-				if (propsSet == (_index | _type)) {
-					StringBuilder buf = new StringBuilder();
-					buf.append("/");
-					SimpleEndpoint.pathEncode(request.index, buf);
```

Comment:
Seems like `_index` should not be dropped

Reply:
The `_index` part is before this if condition. https://github.com/opensearch-project/opensearch-java/pull/132/files/1e01fbeae03463bf2d47b8d395bce3701901448d#diff-2bd95eccdb4274c5118be928fb9f466034f19fc2cdcfc3e5c82609585af94e0aL576

Reply:
Ah ... not visible in changeset, thank you!